style.css

```diff
@@ -82,7 +85,14 @@ a {
   }
 
   .nav-list .close-button {
-    color: #000;
+    background-color: transparent;
+    border: none;
+    color: #fff;
+    cursor: pointer;
+    font-size: 18px;
+    padding: 0;
+    margin-top: 30px;
+    margin-right: 100px;
   }
```

Comment:
- [ ] Nice work implementing the close button :+1: However it would be nice to have it more closer to the right and make it a little be more bigger to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-23 09-28-18](https://github.com/Eccb7/Portfolio2.0/assets/91142432/f419a3f5-9f5a-411a-9ccd-28679c654485)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-05-15 17-26-08](https://github.com/Eccb7/Portfolio2.0/assets/91142432/2d8422c2-48fa-46f9-9875-aaec6ffec2b9)

_**HIT** You can use this code for the close button_
```css
.nav-list .close-button {
    background-color: transparent;
    cursor: pointer;
    font-size: 25px;
    margin: 10px 20px 0 0;
}
```